### PR TITLE
New games - 18.04.2017

### PR DESCRIPTION
**Pull request by odziom91**
List of implemented games:
18 Wheels of Steel: Across America (Offline installer, Not-Free)
Resident Evil 3 (Offline installer, Not-Free)
Civilization V (Steam installer, Not-Free)
Icy Tower 1.5 (Online installer, Free)

### CODE REVIEW

File: Games/18 Wheels of Steel Across America/18WoSAA/script.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+include(["Functions", "QuickScript", "LocalInstallerScript"]);
+
+new LocalInstallerScript()
+    .name("18 Wheels of Steel: Across America")     // name of the application
```
Comment (plata) [inline Games/18 Wheels of Steel Across America/18WoSAA/script.js]: Please remove the comments here (I only used them in the wiki as an explanation).

**Comment by plata** [comment]
Thanks a lot for your submission. It would be great if you could do one pull request per game (makes it easier to track the changes in the repository later on).

**Comment by odziom91** [comment]
Hello :)
Comments in scripts have been removed. Please check new commit.
I will try to make another scripts for more games and they will be uploaded as one pull request per game. :)

**Comment by plata** [comment]
I have updated the [README](https://github.com/PlayOnLinux/Scripts/blob/master/README.md) to be more clear about the pull requests.

Also, I have split your pull request and committed them:
* #198
* #199 
* #200 
* #201

**Pull request closed without merging**